homeassistant/components/light/aurora.py

```diff
@@ -0,0 +1,153 @@
+"""
+Support for Nanoleaf Aurora platform.
+
+Based in large parts upon Software-2's ha-aurora and fully
+reliant on Software-2's nanoleaf-aurora Python Library, see
+https://github.com/software-2/ha-aurora as well as
+https://github.com/software-2/nanoleaf
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/light.nanoleaf_aurora/
```

Comment:
The url is wrong. It should end with the platform name.

Reply:
I don't think I get it. There is no platform for the nanoleaf aurora lights, there is however a aurora sensor which has nothing to do with those lights (which is why I named the documentation light.nanoleaf_aurora) - did I misunderstand something/do something incorrectly?

Reply:
The name of the platform module is `aurora.py`. Then you have to name the docs page `https://home-assistant.io/components/light.aurora/` eg.

Reply:
I get it now, thanks - If I rename it to aurora the component page will display a link to the binary sensor, thinking they are related, is there a way to overwrite that behaviour or do you recon that this is OK? - if not, I will probably have to rename the component itself

Reply:
Yes, we shouldn't have two platforms with the same name that are not connected. Please rename this platform. `nanoleaf_aurora.py` is ok.

Reply:
Thanks, I've renamed it in #13831 , for the others I'll create separate PRs as well, if possible it would be great if this could get in 0.67 so that nobody has to update their config in the version following afterwards